### DISCUSSION TITLE
Great_Expectations Conditional Expectation in Spark 3.2.1 with Pandas API in DataBricks

### DISCUSSION THREAD

**Discussion by fullysane** [great-expectations/great_expectations#5072] [Support] [answered]
We want to implement Great_Expectations' in DataBricks with Conditional Expectation. According to GE's documentation https://docs.greatexpectations.io/docs/reference/expectations/conditional_expectations is only available for Pandas this argument must be set to "pandas" by default, thus, demanding the appropriate syntax. Other engines might be implemented in the future.

Does anyone knows if it can be implemented in Spark 3.2.1 which integrated with Pandas APIs? if not, any suggestion for handle conditional expectation in DataBricks with Spark 3.2.1?

**Comment by austiezr** [comment, marked as answer]
Hey @fullysane ! Thanks for reaching out about this. We do currently have experimental support for conditional expectations on spark; you will need to pass the `condition_parser` as `great_expectations__experimental__`. This is experimental functionality so you may see unexpected behavior from time to time.
